src/boot/efi/linux.c

```diff
@@ -28,21 +29,25 @@ static VOID linux_efi_handover(EFI_HANDLE image, struct boot_params *params) {
         handover(image, ST, params);
 }
 
-EFI_STATUS linux_exec(EFI_HANDLE image,
-                      CHAR8 *cmdline, UINTN cmdline_len,
-                      UINTN linux_addr,
-                      UINTN initrd_addr, UINTN initrd_size) {
+EFI_STATUS linux_exec(
+                EFI_HANDLE image,
+                CHAR8 *cmdline, UINTN cmdline_len,
+                VOID *linux_buffer,
+                VOID *initrd_buffer, UINTN initrd_length) {
```

Comment:
the `initrd_buffer` should probably be const.  And maybe `linux_buffer` too?

Reply:
... and cmdline too. I just added changed the type here and fixed the indention according to the coding style.